Source/WebKit/UIProcess/Automation/WebAutomationSession.cpp

```diff
@@ -375,6 +375,12 @@ Inspector::Protocol::ErrorStringOr<void> WebAutomationSession::closeBrowsingCont
     return { };
 }
 
+Inspector::Protocol::ErrorStringOr<void> WebAutomationSession::deleteSession()
+{
+    terminate();
+    return { };
+}
```

Comment:
It seems win platform tried to solve this, but patched WebAutomationSession::closeBrowsingContext instead, maybe they should be sending this new message instead? I don't think we should do that in this patch, though, not even in a follow up, but let them know we are adding this at least.

Reply:
cc'ing @haruhisa-shin (Author of win bits in #24495 )

Reply:
Thank you for letting me know.
We too will look into whether we can use this new message.

So far, WebDriver of win port seems to work fine after applying this patch.